features/features/package_managers/rebar_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'feature_helper'
+
+describe "Rebar Dependencies" do
+  # As a Node developer
+  # I want to be able to manage NPM dependencies
+
+  let(:erlang_developer) { LicenseFinder::TestingDSL::User.new }
+
+  specify "are shown in reports" do
+    LicenseFinder::TestingDSL::RebarProject.create
+    erlang_developer.run_license_finder
+    expect(erlang_developer).to be_seeing_line "sqerl, \"BRANCH: master\", \"Apache 2.0\""
```

Comment:
I based this off of how other package_managers were implemented, it was just a sample package to ensure that rebar support was working in the integration tests.